docs/content/configuration/storage.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Configuring Storage
+description: Configuring Storage in Athens
+---
+
+## Storage
+
+The Athens proxy supports many storage types:
+
+1. [Memory](#memory)
+1. [Disk](#disk)
+1. [Mongo](#mongo)
+1. [Google Cloud Storage](#google-cloud-storage)
+1. [AWS S3](#aws-s3)
+1. [Minio](#minio)
+1. [DigitalOcean Spaces](#digitalocean-spaces)
+
+All of them can be configured using `config` file. You need to set proper driver in `StorageType` value or you can set it in environment variable `ATHENS_STORAGE_TYPE` on your server.
+Also in most of drivers you need to provide additional configuration data which will be described below.
+
+## Memory
+
+This storage don't need any specific configuration and it's also used by default in Athens project. It writes all of data into local disk into `tmp` dir.
+
+**This storage type should only be used for development purpose!**
+
+##### config:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "memory"
+    
+## Disk
+
+Disk storage allows modules to be stored on a file system. The location on disk where modules will be stored can be configured.
+
+##### config:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "disk"
+    
+    [Storage]
+        [Storage.Disk]
+            RootPath = "/path/on/disk"
+            
+where `/path/on/disk` is your desired location. Also it can be set using `ATHENS_DISK_STORAGE_ROOT` env
+
+## Mongo
+
+This driver is using [Mongo](https://www.mongodb.com/) server as data storage. On start this driver will create `athens` database and `module` collection inside your mongo server.
+
+##### config:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "mongo"
+    
+    [Storage]
+        [Storage.Mongo]
+            # Full URL for mongo storage
+            # Env override: ATHENS_MONGO_STORAGE_URL
+            URL = "mongodb://127.0.0.1:27017"
+    
+            # Not required parameter
+            # Path to certificate to use for the mongo connection
+            # Env override: ATHENS_MONGO_CERT_PATH
+            CertPath = "/path/to/cert/file"
+    
+            # Timeout for networks calls made to Mongo in seconds
+            # Defaults to Global Timeout
+            # Env override: MONGO_CONN_TIMEOUT_SEC
+            Timeout = 300
+    
+            # Not required parameter
+            # Allows for insecure SSL / http connections to mongo storage
+            # Should be used for testing or development only
+            # Env override: ATHENS_MONGO_INSECURE
+            Insecure = false                    
+            
+## Google Cloud Storage
+
+This driver is using [Google Object Storage](https://cloud.google.com/storage/) and assumes that you already have `account` in it. 
```

Comment:
Can you write "This driver uses [Google Cloud Storage](https://cloud.google.com/storage) and assumes you already have an `account` in it."?